stack-windows.yaml

```diff
@@ -35,3 +35,24 @@ extra-deps:
 
 nix:
   packages: [zlib]
+
+configure-options:
+    heapsize:
+        - --disable-library-for-ghci
+        - --disable-library-stripping
+# Otherwise the ghcide will fail with:
+# ```
+# ghcide                           > ghc.exe: unable to load package `heapsize-0.2'
+# ghcide                           > ghc-iserv:  | D:\a\1\s\.stack-work\install\52d658b2\lib\x86_64-windows-ghc-8.10.1\heapsize-0.2-KCPoGpPDcevACNftTTY2at\HSheapsize-0.2-KCPoGpPDcevACNftTTY2at.o: unknown symbol `heap_view_closurePtrs'
+#
+# Cause:
+# The pre-linked object file is missing the heapsize_prim.o symbols table (from the cbits object)
+#
+# Reason: The ld invocation is stripping too much
+#
+# Quoting https://downloads.haskell.org/ghc/latest/docs/html/users_guide/packages.html
+#
+# > To load a package foo, GHCi can load its libHSfoo.a library directly, but it can also load a package in the form of a single HSfoo.o file that has been pre-linked. Loading the .o file is slightly quicker, but at the expense of having another copy of the compiled package. The rule of thumb is that if the modules of the package were compiled with -split-sections then building the HSfoo.o is worthwhile because it saves time when loading the package into GHCi. Without -split-sections, there is not much difference in load time between the .o and .a libraries, so it is better to save the disk space and only keep the .a around. In a GHC distribution we provide .o files for most packages except the GHC package itself.
+# > The HSfoo.o file is built by Cabal automatically; use --disable-library-for-ghci to disable it. To build one manually, the following GNU ld command can be used:
+
+# > ld -r --whole-archive -o HSfoo.o libHSfoo.a
```

Comment:
OMG :facepalm: 

Thanks for the detailed report of the root cause

Reply:
Yeah, but the fix is not enough. There is another stack issue after it.